playbooks/archivematica-bionic/bionic-ci.jenkinsci

```diff
@@ -0,0 +1,176 @@
+node {
+stage('Get code') {
+   git branch: 'stable/1.7.x', poll: false,
+       url: 'https://github.com/artefactual/archivematica'
+
+   git branch: 'stable/0.12.x', poll: false,
+       url: 'https://github.com/artefactual/archivematica-storage-service'
+
+
+   checkout([$class: 'GitSCM',
+            branches: [[name: 'dev/add-jenkins-bionic']],
+            doGenerateSubmoduleConfigurations: false,
+            extensions:
+            [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'deploy-pub']],
+            submoduleCfg: [],
+            userRemoteConfigs: [[url: 'https://github.com/artefactual/deploy-pub']]])
+
+}
+
+stage ('Create vm') {
+sh '''
+   cd deploy-pub/playbooks/archivematica-bionic
+   source ~/.secrets/openrc.sh
+   ansible-galaxy install -f -p roles -r requirements.yml
+   export VAGRANT_VAGRANTFILE=Vagrantfile.openstack
+   vagrant up
+   vagrant ssh-config | tee >( grep HostName  | awk '{print $2}' > $WORKSPACE/.host) \
+                            >( grep User | awk '{print $2}' > $WORKSPACE/.user ) \
+                            >( grep IdentityFile | awk '{print $2}' > $WORKSPACE/.key )
+
+   vagrant ssh -c "sudo ln -sf /home/ubuntu /home/vagrant"
+   vagrant ssh -c "sudo adduser ubuntu archivematica"
+
+'''
+env.SERVER = sh(script: "cat .host", returnStdout: true).trim()
+env.USER = sh(script: "cat .user", returnStdout: true).trim()
+env.KEY = sh(script: "cat .key", returnStdout: true).trim()
+}
+
+
+
+stage('Configure acceptance tests') {
+git branch: 'master', url: 
+'https://github.com/artefactual-labs/archivematica-acceptance-tests'
+   properties([disableConcurrentBuilds(),
+            gitLabConnection(''),
+            [$class: 'RebuildSettings',
+             autoRebuild: false,
+             rebuildDisabled: false],
+            pipelineTriggers([pollSCM('*/5 * * * *')])])
```

Comment:
What's the difference between `git` and `checkout`? Are they being used indistinctly?

Reply:
`git` is a subclass of `checkout`. More info here: https://jenkins.io/doc/pipeline/steps/git/

I had to switch to `checkout` in order to clone to a different folder, but leave the others untouched